### DISCUSSION TITLE
Limit access to OpenID Connect Clients to specific groups

### DISCUSSION THREAD

**Discussion by publicdesert** [authelia/authelia#3588] [Q&A] [answered]
I have multiple services set up to use Authelia's OpenID Connect interface (which works really well!). I was wondering if it is possible (or planned) to limit access to certain OpenID clients only to specific user groups like you can do when using Authelia as middleware. 
Thank you very much!

**Comment by james-d-elliott** [comment, marked as answer]
Sorry for not seeing this earlier, implemented in 4.38 I believe.
